diracos/scriptTemplates/bundle_diracos_script_tpl.sh

```diff
@@ -32,19 +32,23 @@ cp -r /tmp/pipDirac/bin/ $DIRACOS/usr/
 
 # Fix the shebang for python
 echo "Fixing the shebang"
-grep -rIl '#!/usr/bin/python' /tmp/diracos | xargs sed -i 's:#!/usr/bin/python:#!/usr/bin/env python:g'
+grep -rIl '#!/usr/bin/python' $DIRACOS | xargs sed -i 's:#!/usr/bin/python:#!/usr/bin/env python:g'
+
+# Fix RPATHs
+set -x
+export CONDA_BASE_TMP=$(mktemp -d)
+(cd "${CONDA_BASE_TMP}" &&
+ curl -LO https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh &&
+ bash Miniconda3-latest-Linux-x86_64.sh -b -p "${CONDA_BASE_TMP}/miniconda")
```

Comment:
Can we somehow not use latest, since this is a bit problematic for reproducibility?

Reply:
And also this needs some explanation in the documentation